spec/lib/schedule_spec.rb

```diff
@@ -51,19 +51,32 @@
       subject.dates(start_date).first.should == start_date + 4 # Saturday
 
       start_date = Date.new(2014, 8, 20) # Wednesday
-      subject.dates(start_date).first.should == start_date + 6 # Tuesday
+      subject.dates(start_date).first.should == start_date + 7 # Wednesday
 
       start_date = Date.new(2014, 8, 21) # Thursday
-      subject.dates(start_date).first.should == start_date + 6 # Wednesday
+      subject.dates(start_date).first.should == start_date + 7 # Thursday
 
       start_date = Date.new(2014, 8, 22) # Friday
-      subject.dates(start_date).first.should == start_date + 6 # Thursday
+      subject.dates(start_date).first.should == start_date + 7 # Friday
 
       start_date = Date.new(2014, 8, 23) # Saturday
-      subject.dates(start_date).first.should == start_date + 5 # Thursday
+      subject.dates(start_date).first.should == start_date + 6 # Friday
 
       start_date = Date.new(2014, 8, 24) # Sunday
-      subject.dates(start_date).first.should == start_date + 4 # Thursday
+      subject.dates(start_date).first.should == start_date + 5 # Friday
+    end
+
+    it "assumes bookings are not processed on bank holidays" do
+      #      April 2015
+      # Su Mo Tu We Th Fr Sa
+      #           1  2 <3> 4
+      #  5 <6> 7  8  9 10 11
+      # 12 13 14 15 16 17 18
+      # 19 20 21 22 23 24 25
+      # 26 27 28 29 30
+
+      start_date = Date.new(2015, 4, 2)
+      subject.dates(start_date).first.should == start_date + 8
     end
```

Comment:
Someone runs `rake update_bank_holidays` in the future, and that gives us a new `bank-holidays.json`. https://www.gov.uk/bank-holidays.json [isn't a supported API](https://insidegovuk.blog.gov.uk/2014/09/15/current-state-of-apis-on-gov-uk/) from GOV.UK, though, so they could well remove all past bank holidays from it with no notice.

Having these tests depend (albeit indirectly) on an unsupported API seems worrisome, particularly given we could just mock the data for this test (thus decoupling the tests from what is, really, live config). It feels, I suppose, like we're testing the data as much as the behaviour, rather than testing the latter in isolation.

We should probably have another test checking that the data is still in the format we want, too, but I can't find a JSON schema that's published anywhere for us to do that (which, I guess, ties back into the fact that it's an unsupported API)…